Project.toml

```diff
@@ -10,21 +10,4 @@ UUIDs = "cf7118a7-6976-5b1a-9a39-7adc72f591a4"
 
 [compat]
 CodeTracking = "0.5.9, 1"
-julia = "1"
-
-[extras]
-DataFrames = "a93c6f00-e57d-5684-b7b6-d8193f3e46c0"
-Dates = "ade2ca70-3891-5945-98fb-dc099432e06a"
-Distributed = "8ba89e20-285c-5b6f-9357-94700520ee1b"
-HTTP = "cd3eb016-35fb-5094-929b-558a96fad6f3"
-LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
-Mmap = "a63ad114-7e13-5084-954f-fe012c677804"
-PyCall = "438e738f-606a-5dbb-bf0a-cddfbfd45ab0"
-SHA = "ea8e919c-243c-51af-8825-aaa63cd721ce"
-SparseArrays = "2f01184e-e22b-5df5-ae63-d93ebab69eaf"
-TableReader = "70df011a-6618-58d7-8e16-3cf9e384cb47"
-Tensors = "48a634ad-e948-5137-8d70-aa71f2a747f4"
-Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
-
-[targets]
-test = ["Test", "Dates", "Distributed", "HTTP", "LinearAlgebra", "Mmap", "PyCall", "SHA", "SparseArrays", "Tensors", "TableReader", "DataFrames"]
+julia = "1.6"
```

Comment:
Can we keep this? I don't see any advantage to the `test/Project.toml` approach. In my opinion the other way is very strange right now since it isn't a proper project.

Reply:
Well, isn't `test/Project.toml` more "up-to-date" way to describe test-specific dependencies as described in: <https://pkgdocs.julialang.org/v1/creating-packages/#Test-specific-dependencies-in-Julia-1.2-and-above>?
I'm actually very indifferent between those two though.

> In my opinion the other way is very strange right now since it isn't a proper project.
Hm, I couldn't get the point. Any directory is a proper project as far as it contains `Project.toml`?

Reply:
It gives you a false sense that you can activate the test project and then include the `runtests.jl`. That's why I'm saying it is but a true project since you would never activate it. It is purely a list of test dependencies, so what's the advantage of putting that in a separate file?

Reply:
I see. Then if we use `test/Project.toml` approach, you think it should include the package itself (and thus `test/Manifest.toml` also needs to be checked-in), right?
Now I'd like to just keep test dependencies in `Project.toml`, and simply wonder what situations `test/Project.toml` is preferred.

Reply:
> Now I'd like to just keep test dependencies in Project.toml, and simply wonder what situations test/Project.toml is preferred

Right now, I would say never. Something like https://github.com/JuliaLang/Pkg.jl/issues/1233 is needed to make these first-class to be useful. In my opinion.

Reply:
Okay, thanks for your explanation!